grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -743,6 +743,68 @@ class ProductAvailabilityService {
         return availableBinLocations
     }
 
+    AvailableItem getAvailableItemWithDefaultLots(Location origin, String productCode) {
```

Comment:
I think the name of this method doesn't fit to what the method is actually doing. We should either think about the name again, or at least add a comment above the method.
My point is that if you find 3 items, in fact there is 3 available items found, but the method would return `null`, which is a bit misleading.